client/src/main/java/io/pravega/client/segment/impl/SegmentMetadataClientImpl.java

```diff
@@ -115,7 +115,8 @@ RawClient getConnection() {
         log.debug("Getting segment info for segment: {}", segmentId);
         RawClient connection = getConnection();
         long requestId = connection.getFlow().getNextSequenceNumber();
-        return connection.sendRequest(requestId, new GetStreamSegmentInfo(requestId, segmentId.getScopedName(), delegationToken))
+        return connection.sendRequest(requestId,
+                                      new GetStreamSegmentInfo(requestId, segmentId.getScopedName(), delegationToken))
```

Comment:
I don't see any functional change in this file.

Reply:
Yes, that's right. I was doing some functional changes in that file at first but decide to not use that change in the end. The modified line was a little long so I make it shorter. I think it might be useful so I keep it there.